.github/workflows/ci.yaml

```diff
@@ -323,6 +323,36 @@ jobs:
         with:
           message-path: download-link.txt
 
+  e2e-light:
```

Comment:
is this just e2e tests run inside a docker container?
If so, a comment would be great